### PR TITLE
Chat back

### PR DESCRIPTION
**Pull request by bsibiya**
Note: Edited the main **index.css**

### CODE REVIEW

File: src/containers/chat/index.js [hunk 1]
```diff
@@ -3,11 +3,24 @@ import { Route } from 'react-router-dom'
...
+import './index.css'
 const Chat = () => (
-    <div>
-        <p>Chat</p>
-    </div>
+ <div className="msgContainer">
+     <Account_Screen />
+     <div>
+           <div className="oval-2">
+           <hr className="line"/>
```
Comment (NardusVanDerVyver) [inline src/containers/chat/index.js]: The indentation here needs to be adjusted.

File: src/containers/chat/index.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    width: 717px;
+    width: 50%;	
+    border-radius: 24px;	
+    background-color: #EFF2F3;
+}
+html {
+    background-color: #ffffff;
+}
+.line-copy {
+	box-sizing: border-box;
```
Comment (NardusVanDerVyver) [inline src/containers/chat/index.css]: Indentation needs to be consistent with the rest of the file.

**Review by NardusVanDerVyver — CHANGES_REQUESTED** [review]
Code was reviewed

File: src/containers/chat/account_screen/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Route } from 'react-router-dom'
+import Login from '../../login'
+import Register from '../../register'
+
+import './index.css'
+
+const Account_Screen = () => (
+<div class="chatbox">
+    <div className="incoming">
```
Comment (ReinhardtCromhout) [inline src/containers/chat/account_screen/index.js:10]: Give html and css classes proper names. Giving a div a class name of "msgBox1" for example does not make sense. What does the 1 mean. How does "msgBox1" differ from "msgBox2". Why are they styled differently?

**Review by ReinhardtCromhout — CHANGES_REQUESTED** [review]

**Review by jaderabbit — CHANGES_REQUESTED** [review]
Look at @ReinhardtCromhout comments

File: debug.log [hunk 4]
```diff
@@ -0,0 +1 @@
+[0108/065826.939:ERROR:registration_protocol_win.cc(84)] TransactNamedPipe: The pipe has been ended. (0x6D)
```
Comment (jaderabbit) [inline debug.log]: these should be in `.gitignore`? Why is this here?

**Review by NardusVanDerVyver — DISMISSED** [review]

File: src/containers/chat/Existing_chat/index.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```
Comment (jaderabbit) [inline src/containers/chat/Existing_chat/index.js:1]: New_Chat versus Existing_chat

Pick a filenaming convention and stick to it

File: src/containers/chat/New_Chat/index.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (jaderabbit) [inline src/containers/chat/New_Chat/index.js]: what is the difference between New_Chat and Existing_chat?
Surely this should be a single `Chat` component with options?

File: src/containers/chat/index.css [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+	height: 2.28px;
+	width: 18px;
+	border: 3px solid #979797;
+}
+
+.oval-2 {
+	height: 54px;
+	width: 54px;
+    background-color: #D8D8D8;
```
Comment (jaderabbit) [inline src/containers/chat/index.css]: indentation?!

File: src/containers/chat/index.js [hunk 8]
```diff
@@ -1,12 +1,30 @@
...
+import './index.css'
+import Button from 'material-ui/Button/Button';
 
 const Chat = () => (
     <div>
-        <p>Chat</p>
+        <div className="msgContainer">
+            <Account_Screen />   
+        </div>
+        <div>
```
Comment (jaderabbit) [inline src/containers/chat/index.js]: what is that div for? surely it's just polluting the code base?

**Review by jaderabbit — CHANGES_REQUESTED** [review]
Code needs to be cleaner. 
Indentation
Naming
More thought about it

File: src/containers/chat/index.js (hunk 8 above)
Comment (ReinhardtCromhout) [reply src/containers/chat/index.js]: Please resolve this before the PR can be approved.

**Review by ReinhardtCromhout — CHANGES_REQUESTED** [review]

File: src/containers/chat/index.css [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+.msgInput {
+    height: 43px;	
+    width: 70%;
+    border-radius: 24px;	
+	background-color: #EFF2F3;
+	margin-left: -5px;
+    margin-top: 5px;
+}
+
+.inputSpace {
```
Comment (jaderabbit) [inline src/containers/chat/index.css]: please decide between camelcase and dash case

**Review by jaderabbit — CHANGES_REQUESTED** [review]

File: src/containers/chat/index.css [hunk 10]
```diff
@@ -39,7 +39,7 @@ html {
 .oval-2 {
```
Comment (jaderabbit) [inline src/containers/chat/index.css]: why is it called `oval-2`?

File: .gitignore [hunk 11]
```diff
@@ -19,3 +19,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+debug.log*
```
Comment (jaderabbit) [inline .gitignore:22]: :tada:

**Review by NardusVanDerVyver — APPROVED** [review]

File: debug.log (hunk 4 above)
Comment (jaderabbit) [reply debug.log]: why is this still here? manually remove from git then it should be ignored

File: src/containers/chat/index.css [hunk 12]
```diff
@@ -0,0 +1,52 @@
+.message-container {
+    padding: 10px;
```
Comment (jaderabbit) [inline src/containers/chat/index.css]: fix indentation. Install a linter. I think you're mixing up tabs and spaces

File: src/containers/chat/index.js [hunk 13]
```diff
@@ -1,12 +1,30 @@
 import React from 'react';
-import { Route } from 'react-router-dom'
-import Login from '../login'
-import Register from '../register'
 import Body from '../body'
+import Account_Screen from './account_screen'
+import Existing_Chat from './existing_chat'
```
Comment (jaderabbit) [inline src/containers/chat/index.js]: What did we resolve about New_Chat and Existing_Chat? Why do you have both?

**Review by jaderabbit — CHANGES_REQUESTED** [review]

File: src/containers/chat/index.js [hunk 14]
```diff
@@ -1,12 +1,28 @@
...
+import { withStyles } from 'material-ui/styles';
+import Paper from 'material-ui/Paper';
+import Grid from 'material-ui/Grid';
+import './index.css'
+import Button from 'material-ui/Button/Button';
 
 const Chat = () => (
-    <div>
-        <p>Chat</p>
+    <div className="main-container">
```
Comment (jaderabbit) [inline src/containers/chat/index.js:13]: :tada:

**Review by jaderabbit — APPROVED** [review]
